File: java/client/src/org/openqa/selenium/devtools/page/model/FontSizes.java
package org.openqa.selenium.devtools.page.model;

import org.openqa.selenium.Beta;
import org.openqa.selenium.json.JsonInput;

/**
 * Default font sizes.EXPERIMENTAL
 */
@Beta
public class FontSizes {

  /**
   * Default standard font size.
   */
  private final Integer standard;
  /**
   * Default fixed font size.
   */
  private final Integer fixed;

  public FontSizes(Integer standard, Integer fixed) {
    this.standard = standard;
    this.fixed = fixed;
  }

  private static FontSizes fromJson(JsonInput input) {
    Integer standard = null, fixed = null;
    input.beginObject();
    while (input.hasNext()) {
      switch (input.nextName()) {
        case "standard":
          standard = input.read(Integer.class);
          break;
        case "fixed":
          fixed = input.read(Integer.class);
          break;
        default:
          input.skipValue();
          break;
      }
    }
    input.endObject();
    return new FontSizes(standard, fixed);
  }
}

File: dotnet/src/webdriver/DevTools/AutoGenerated/Page/FontSizes.cs
namespace OpenQA.Selenium.DevTools.Page
{
    using Newtonsoft.Json;

    /// <summary>
    /// Default font sizes.
    /// </summary>
    public sealed class FontSizes
    {
        /// <summary>
        /// Default standard font size.
        ///</summary>
        [JsonProperty("standard", DefaultValueHandling = DefaultValueHandling.Ignore)]
        public long? Standard
        {
            get;
            set;
        }
        /// <summary>
        /// Default fixed font size.
        ///</summary>
        [JsonProperty("fixed", DefaultValueHandling = DefaultValueHandling.Ignore)]
        public long? Fixed
        {
            get;
            set;
        }
    }
}